### PR TITLE
Add missing types from form schema, don't require autofill conditions

### PR DESCRIPTION
**Pull request by gigxz**
[//]: # 'remove this if PR is for a release-* branch'
# _Please squash merge this PR_

## Description

1. add group components that were missing from schema
2. don't require conditions on autofill. We have forms that have autofills that should always run, for example when summing up dependent fields. Currently those forms set dummy values. Update the schema to just allow them to be nullable. I kept the GraphQL schema as-is and added default_value behavior, so as not to cause a breaking change. This does not affect behavior, because the frontend already treats `autofillBehavior: 'ANY'. autofillWhen: []` as "always autofill"

## Type of change
- [x] Bug fix

## Checklist before requesting review
- [x] I have performed a self-review of my code
- [x] I have run the code that is being changed under ideal conditions, and it doesn't fail
- [x] My code includes comments and/or descriptive variable names to help other engineers understand the intent (or not applicable)
- [x] My code follows the style guidelines of this project (rubocop)
- [x] I have updated the documentation (or not applicable)
- [x] If it's not obvious how to test this change, I have provided testing instructions in this PR or the related issue

### CODE REVIEW

File: drivers/hmis/app/graphql/types/forms/autofill_value.rb [hunk 1]
```diff
@@ -19,9 +19,9 @@ class Forms::AutofillValue < Types::BaseObject
     field :sum_questions, [String], 'Link IDs of numeric questions to sum up and set as the value if condition is met', null: true
     field :formula, String, 'Expression with mathematical or logical function defining the value', null: true
 
-    # Condition specifying when to perform this autofill
-    field :autofill_behavior, Types::Forms::Enums::EnableBehavior, null: false
-    field :autofill_when, [Types::Forms::EnableWhen], null: false
+    # Condition specifying when to perform this autofill. If not provided, the autofill will always run.
+    field :autofill_behavior, Types::Forms::Enums::EnableBehavior, null: true
+    field :autofill_when, [Types::Forms::EnableWhen], null: true
```
Comment (gigxz) [inline drivers/hmis/app/graphql/types/forms/autofill_value.rb]: actually this could cause a bug when deployed without the frontend change, hold off

Comment (gigxz) [reply drivers/hmis/app/graphql/types/forms/autofill_value.rb]: https://github.com/greenriver/hmis-frontend/pull/843

**Comment by gigxz** [comment]
this could be re targeted to 124

**Review by martha — APPROVED** [review]

**Review by ttoomey — APPROVED** [review]
looks good, sorry I overlooked those component types

**Pull request merged**